test/Select-test.js

```diff
@@ -1047,7 +1047,7 @@ describe('Select', () => {
 	describe('with allowCreate=true', () => {
 
 		// TODO: allowCreate hasn't been implemented yet in 1.x
-		return;
+		// return;
```

Comment:
I guess you can just remove the line completely, and the TODO comment as well :)